chapter03/dtj/ex1

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+print "Enter some input, end with ctrl-d / ctrl-z\n";
+
+chomp(my @input = <STDIN>);
```

Comment:
it's a bit vague but I think the newlines are meant to be preserved, then you wouldn't need the foreach loop below.